src/pages/index.tsx

```diff
@@ -22,8 +23,10 @@ const Home: NextPage = () => {
   return (
     <HomeLayout>
       <Header />
+      <Element name="cards">
```

Comment:
Do we need this if you're just using scrollTo a pixel height?

Reply:
no